code/datums/mind.dm

```diff
@@ -1442,7 +1442,7 @@
 			SSjob.SendToLateJoin(current)

```

Comment:
This is the proc that is sending and buckling them to the arrivals shuttle in the first place. Why is it even running?

Reply:
It's also called in makebody helper. I didn't check if it's actually needed , might not be.

Reply:
It's there as a safety incase the game can't find any valid wizardstart locations.